runtime/onert/core/onert-plugin.pc.in

```diff
@@ -0,0 +1,10 @@
+prefix=@CMAKE_INSTALL_PREFIX@
+libdir=${prefix}/@ONERT_INSTALL_COREDIR@
+includedir=${prefix}/@CMAKE_INSTALL_INCLUDEDIR@
+version=@CMAKE_PROJECT_VERSION@
+
+Name: onert-plugin
+Description: onert plugin API
+Version: @CMAKE_PROJECT_VERSION@
+Libs: -L@libdir@ -lonert_core
+Cflags: -I@includedir@/onert
```

Comment:
Is it OK to change the include directory from nnfw to onert?

Reply:
It is not changing directory. `onert-plugin` had both `-I@includedir@/onert` and `-I@includedir@/nnfw` but `-I@includedir@/nnfw` is removed on this PR because it is not used for core header.